src/main/java/com/epam/preprod/sirenko/CreatedByInputProductImpl.java

```diff
@@ -0,0 +1,126 @@
+package com.epam.preprod.sirenko;
+
+import com.epam.preprod.sirenko.enums.PetGroup;
+import com.epam.preprod.sirenko.enums.Season;
+import com.epam.preprod.sirenko.enums.Size;
+import com.epam.preprod.sirenko.util.ConsoleReader;
+import com.epam.preprod.sirenko.util.PrintToConsole;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+
+public class CreatedByInputProductImpl implements Strategy {
+	private static final String INPUT_PRICE = "Enter price in format 123.04 and press Enter";
+	private static final String INPUT_WEIGHT = "Enter weight in format 200 and press Enter";
+	private static final String INPUT_STRING_NAME = "Enter name from 5 to 15 letters only and press Enter";
+	private static final String INPUT_STRING_BRAND_NAME = "Enter brand name from 5 to 15 letters only and press Enter";
+	private static final String INPUT_PET_GROUP = "Enter pet group name and press Enter";
+	
+	@Override
+	public PetGroup getPetGroup() {
+		PetGroup petGroup = null;
+		for (PetGroup element : PetGroup.values()) {
```

Comment:
We can move it to separate method for better readability

Reply:
done